### ISSUE TITLE
measureText: Support word-break CSS

### ISSUE THREAD

**Issue by sdemjanenko** [Flipboard/react-canvas#73]
It would be awesome if all options are supported.

**Comment by mjohnston** [comment]
Yes, agree. There are a couple PRs open for improved line breaking that I've simply been too busy to look into merging. Will leave this issue open for now.
